From 6b1b4456fa1699a89139c0867fea197557ad24fa Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.i.king@gmail.com>
Date: Mon, 4 Jul 2022 19:53:19 +0100
Subject: [PATCH 12/32] fs/ntfs3: Remove duplicated assignment to variable r

The assignment to variable r is duplicated, the second assignment
is redundant and can be removed.

Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Signed-off-by: Konstantin Komarov <almaz.alexandrovich@paragon-software.com>
---
 fs/ntfs3/run.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fs/ntfs3/run.c b/fs/ntfs3/run.c
index 7609d45a2..aba8ab1b9 100644
--- a/fs/ntfs3/run.c
+++ b/fs/ntfs3/run.c
@@ -563,7 +563,6 @@ bool run_insert_range(struct runs_tree *run, CLST vcn, CLST len)
 	e = run->runs + run->count;
 	r = run->runs + index;
 
-	r = run->runs + index;
 	if (vcn > r->vcn)
 		r += 1;
 
-- 
2.37.1.377.g679aad9e82

